### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by kirtiv1**

### CODE REVIEW

File: images/install_greenplum.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (dimaspivak) [inline images/install_greenplum.sh:15]: I'd put a shebang here (e.g. `#!/usr/bin/env bash`)

Comment (dimaspivak) [reply images/install_greenplum.sh:15]: Should we put a license header here, too, as in some of the other files?

File: start.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                            image=node_image,
+                            volumes=volumes)
+                       for hostname in args.secondary_nodes]
+
+    nodes = [primary_node] + secondary_nodes
+    cluster = Cluster(*nodes)
+    cluster.primary_node = primary_node
+    cluster.start(args.network, pull_images=args.always_pull)
+
+    primary_node.put_file(HOST_FILE_PATH, '\n'.join(args.secondary_nodes))
```
Comment (dimaspivak) [inline start.py:79]: Can you add comments here and in a few other places below to give someone with no clue what's being done a little bit of a clue? :)

**Review by dimaspivak — CHANGES_REQUESTED** [review]
Pretty much just nits. Great work!

File: images/install_greenplum.sh (hunk 1 above)
Comment (kirtiv1) [reply images/install_greenplum.sh:15]: Done.

File: start.py (hunk 2 above)
Comment (kirtiv1) [reply start.py:79]: Done.

**Review by dimaspivak — APPROVED** [review]

**Pull request merged**